yoda-toolfile.spec

```diff
@@ -15,7 +15,9 @@ cat << \EOF_TOOLFILE >%i/etc/scram.d/yoda.xml
     <environment name="LIBDIR" default="$YODA_BASE/lib"/>
     <environment name="INCLUDE" default="$YODA_BASE/include"/>
   </client>
-  <use name="boost"/>
+  <use name="python"/>
+  <use name="cython"/>
+  <use name="root_cxxdefaults"/>
```

Comment:
Well, having `root_cxxdefaults` as the last is good, but you also need to specify which part of ROOT is being used. Look at yoda logs and figure out what exactly is being used from ROOT.